src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -2424,11 +2426,25 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
     set_node_expression_string  <- inputs.set_node_expression.map(|(id,expr)| (*id,expr.code.clone()));
     out.source.node_expression_set <+ set_node_expression_string;
 
+    }
+
+
+    // === Set Node Error ===
+    frp::extend! { network
+
+    eval inputs.set_node_error_status([model]((node_id, error)) {
+        if let Some(node) = model.nodes.get_cloned_ref(node_id) {
+            node.set_error_status.emit(error)
+        }
+    });
```

Comment:
Try not using `emit` fro inside `eval`. Currently, our FRP primitives are so rich, that it is not needed almost ever. Just use `<= ... . map(... model.nodes.get_cloned_ref(node_id))` and then zip it with error, and then `node.set_error_status <+ ...`

Reply:
I'm not sure how you mean. Nodes can't be passed through the FRP network as they don't have a default value, so we need to unwrap in normal rust code.